oap-server/server-receiver-plugin/envoy-metrics-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/envoy/als/K8sALSServiceMeshHTTPAnalysis.java

```diff
@@ -39,12 +63,76 @@
 public class K8sALSServiceMeshHTTPAnalysis implements ALSHTTPAnalysis {
     private static final Logger logger = LoggerFactory.getLogger(K8sALSServiceMeshHTTPAnalysis.class);
 
+    private final AtomicReference<Map<String, ServiceMetaInfo>> ipServiceMap = new AtomicReference<>();
+
+    private final ScheduledExecutorService executorService = Executors.newScheduledThreadPool(1, new ThreadFactoryBuilder()
+            .setNameFormat("load-pod-%d").setDaemon(true).build());
+
     @Override public String name() {
         return "k8s-mesh";
     }
 
     @Override public void init(EnvoyMetricReceiverConfig config) {
-        //TODO: Start k8s metadata query timer.
+        executorService.scheduleAtFixedRate(() -> {
```

Comment:
How about we do blocked load at startup, in case als log comes just after startup.

What do you think?